test/functionalTest/harnessFunctions.sh

```diff
@@ -289,9 +290,10 @@ function brokerStartAwait
 
   sleep .5
 
-  # Check that CB started fine
-  curl -s localhost:${port}/version | grep version > /dev/null
+  # Check that CB started
+  curl -s localhost:${port}/version | grep version >> /tmp/brokerStart
```

Comment:
Yes, just some more info in /tmp/brokerStart now that we have the file ...
Without this change I would have had to add some other line to indicate that the broker actually responds.